Yosemite/Yosemite/Model/Mocks/Helpers/ProductStockStatus+Mocks.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+extension ProductStockStatus {
+    public static func from(quantity: Int64) -> ProductStockStatus {
+        switch quantity {
+            case Int64.min ..< 0:
+                return .onBackOrder
+            case 0:
+                return .outOfStock
+            case 1...Int64.max:
+                return .inStock
+            default:
+                preconditionFailure("Unable to parse quantity")
```

Comment:
nit: you can leave out the extreme bounds. I find it clearer that way ;)

```suggestion
            case ..<0:
                return .onBackOrder
            case 0:
                return .outOfStock
            case 1...:
                return .inStock
            default:
                preconditionFailure("Unable to parse quantity")
```